### PR TITLE
Tperps 882 general UI fix

### PR DESCRIPTION
**Pull request by stevenyyan**
### Motivation

Fix styles to match XD designs

### Changes

-

### CODE REVIEW

File: components/General/index.tsx [hunk 1]
```diff
@@ -23,32 +23,31 @@ export const Box = styled.div`
     padding: 12px;
 `;
 
-export const Button = styled.button`
+export const Button = styled.button<{ height?: 'medium' | 'small' | 'extra-small' }>`
```
Comment (dospore) [inline components/General/index.tsx:26]: Can you add a default props here. Maybe medium?

Comment (stevenyyan) [reply components/General/index.tsx:26]: can do, but not giving it a prop would just be equivalent to not setting a height

Comment (stevenyyan) [reply components/General/index.tsx:26]: and my thought was that for any template component we'd avoid giving it preset attributes

Comment (dospore) [reply components/General/index.tsx:26]: Thats a fair point. Can leave it

File: components/Trade/Advanced/TradingPanel/Account/index.tsx [hunk 2]
```diff
@@ -165,7 +91,7 @@ const AccountPanel: React.FC<{
         <AccountInfo zeroBalance={balances.quote.eq(0)}>
             <Title hide={!!order?.exposureBN.toNumber() ?? false}>
                 Margin Account
-                <CalculatorButton onClick={() => showCalculator(true)}>Calculator</CalculatorButton>
+                <Button onClick={() => showCalculator(true)}>Calculator</Button>
```
Comment (dospore) [inline components/Trade/Advanced/TradingPanel/Account/index.tsx:94]: Removing the styles from this cause some weird behaviour on smaller screens 
image

File: components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx [hunk 3]
```diff
@@ -116,36 +125,19 @@ const MarketSelectDropdownButton: React.FC<MarketSelectDropdownButtonProps> = st
...
 const MarketContainer = styled.div`
     font-size: var(--font-size-medium);
-    letter-spacing: -0.4px;
+    letter-spacing: var(--letter-spacing-extra-small);
     display: flex;
     height: var(--height-small-container);
 `;
 
 const SBox = styled(Box)<{
     $display: boolean;
```
Comment (dospore) [inline components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx]:
```suggestion
    $display: boolean;
    color: string;
```

File: components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx [hunk 4]
```diff
@@ -116,36 +125,19 @@ const MarketSelectDropdownButton: React.FC<MarketSelectDropdownButtonProps> = st
...
 }>`
-    background-color: ${(props) => props.color as string}!important;
+    background-color: ${(props: any) => props.color as string}!important;
```
Comment (dospore) [inline components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx]: if there is something in props add it. You will have to check if my suggestion actually works

File: components/Portfolio/Trading/Position/index.tsx [hunk 5]
```diff
@@ -176,7 +175,7 @@ const Position: React.FC<{
                 </TableBody>
             </Table>
             <div className="flex mt-8 justify-center">
-                <Button theme={largeButton} onClick={(e) => onClick(e)}>
+                <Button theme={largeButton} onClick={(e: any) => onClick(e)}>
```
Comment (dospore) [inline components/Portfolio/Trading/Position/index.tsx]: Do button themes still exist here

**Review by dospore — CHANGES_REQUESTED** [review]

File: components/Portfolio/Trading/Position/index.tsx (hunk 5 above)
Comment (stevenyyan) [reply components/Portfolio/Trading/Position/index.tsx]: they don't, there's a lot we need to update for the other portfolio tabs

File: components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx (hunk 3 above)
Comment (stevenyyan) [reply components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx]: added

File: components/Portfolio/Trading/Position/index.tsx (hunk 5 above)
Comment (dospore) [reply components/Portfolio/Trading/Position/index.tsx]: If they arent used we should remove them as we find them

**Review by dospore — DISMISSED** [review]

**Review by dospore — APPROVED** [review]

**Pull request merged**